vnext/JSI/Shared/NapiJsiV8RuntimeHolder.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "NapiJsiV8RuntimeHolder.h"
+
+// V8-JSI
+#include <V8JsiRuntime.h>
+
+using namespace facebook::jsi;
+using namespace facebook::react;
+
+using std::shared_ptr;
+using std::unique_ptr;
+
+namespace Microsoft::JSI {
+
+// See napi_ext_schedule_task_callback definition.
+/*static*/ void NapiJsiV8RuntimeHolder::ScheduleTaskCallback(
+    napi_env env,
+    napi_ext_task_callback taskCb,
+    void *taskData,
+    uint32_t delayMs,
+    napi_finalize finalizeCb,
+    void *finalizeint) {
+
+  NapiJsiV8RuntimeHolder *holder;
+  auto result = napi_get_instance_data(env, (void **)&holder);
+  if (result != napi_status::napi_ok) {
+    //TODO: Signal error
+    return;
+  }
+
+  //TODO: ABI-safe?
```

Comment:
>ABI-safe? [](http://example.com/codeflow?start=10&length=9)

I think we want to call taskCb and pass taskData to it instead (casting into a C++ type is indeed ABI-unsafe).

Reply:
@vmoroz has solved this via a wrapping class. It has been integrated into this branch.